kubernetes/testing_secrets.yml

```diff
@@ -0,0 +1,23 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
I have a slight preference for encrypting these one by one and embedding them alongside other vars so it's easier to see what's changed...is there a reason you prefer it this way?

https://docs.ansible.com/ansible/latest/user_guide/vault.html#use-encrypt-string-to-create-encrypted-variables-to-embed-in-yaml